content/admin/overview/system-overview.md

```diff
@@ -82,7 +82,10 @@ For more information, see "[Configuring backups on your appliance](/admin/config
 
 {% data variables.product.product_name %} is provided as an appliance, and many of the operating system packages are modified compared to the usual Debian distribution. We do not support modifying the underlying operating system for this reason (including operating system upgrades), which is aligned with the [{% data variables.product.prodname_ghe_server %} license and support agreement](https://enterprise.github.com/license), under section 11.3 Exclusions.
 
-Currently, the base operating system for {% data variables.product.product_name %} is Debian 9 (Stretch), which receives support under the Debian Long Term Support program.  There are plans to move to a newer base operating system before the end of the Debian LTS period for Stretch.
+{% ifversion ghes > 3.1 %}
+Currently, the base operating system for {% data variables.product.product_name %} is Debian 10 (Buster), which receives support under the Debian Long Term Support program. 
+
+{% endif %}
```

Comment:
Thanks for opening this @Maya-ross! We can actually remove this versioning because updating the repo only updates supported versions, which is currently 3.3+. (To update earlier versions, we have to manually edit the HTML.) Our repo is currently frozen until January 3rd, but we'll get this merged then!

```suggestion
Currently, the base operating system for {% data variables.product.product_name %} is Debian 10 (Buster), which receives support under the Debian Long Term Support program. 
```